Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/diagnosiskeyprovider/DiagnosisKeyProvider.kt

```diff
@@ -0,0 +1,25 @@
+package de.rki.coronawarnapp.nearby.modules.diagnosiskeyprovider
+
+import com.google.android.gms.nearby.exposurenotification.ExposureConfiguration
+import java.io.File
+
+interface DiagnosisKeyProvider {
```

Comment:
why is this called a provider? this class takes something / "consumes" it and does not provide something.

Reply:
Hmm, it's the interface that the class wrapping `GoogleENFClient.provideDiagnosisKeys` has to implement. So while it does both, it is more responsible for "providing keys into the ENF"  than consuming from our end. :thinking: